configure.ac

```diff
@@ -512,23 +512,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
+	unused
 	shadow
 	missing-prototypes
 	missing-declarations
+	suggest-attribute=noreturn
+	suggest-attribute=format
```

Comment:
Like the above, this suggests call sites where the compiler thinks that a function is a good candidate for an attribute that will enable additional optimizations or features. In this case, it allows for a function accepting variable arguments, with the first argument being a string type, to be processed for security and/or compatibility concerns against a format string.

E.g.

foo(char *, ...)

if you call
foo("%s", (int)5);

gcc will emit a warning that the format doesn't match the parameter.